apps/package.json

```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "lint": "eslint --ext .js,.jsx src test",
-    "test": "npm run lint && grunt test",
+    "test": "./test-low-memory.sh",
     "test-low-memory": "./test-low-memory.sh",
```

Comment:
Next step (after this pr): Replace everything that calls `(npm/yarn) run test-low-memory` with `yarn test`